ruma-federation-api/src/openid.rs

```diff
@@ -0,0 +1,3 @@
+//! OpenID endpoints.
+
+pub mod get_userinfo;
```

Comment:
We like to make the leaf module names for endpoints specific (even if the parent name is redundant). Could you add `openid` to the module name?

```suggestion
pub mod get_openid_userinfo;
```

Reply:
I wonder whether that is something we should look into changing in the future :thinking: